src/pydvl/value/games.py

```diff
@@ -92,13 +92,13 @@ class DummyModel(SupervisedModel):
     def __init__(self) -> None:
         pass
 
-    def fit(self, x: NDArray, y: NDArray) -> None:
```

Comment:
Shouldn't this module moved to the new valuation package?

P.S. This comment is not about this specific line. I just found it easier to comment on it.